web/api/octoprint_messages.py

```diff
@@ -40,6 +43,22 @@ def process_octoprint_status(printer: Printer, status: Dict) -> None:
 
     channels.send_status_to_web(printer.id)
 
+    op_event = status.get('octoprint_event', {})
```

Comment:
Shouldn't we put this block of code in https://github.com/TheSpaghettiDetective/TheSpaghettiDetective/blob/master/web/app/models.py#L575 ? I suspect it's currently send duplicate messages.

Reply:
It does not send duplicate messages. If you see duplicates, then v1 is still enabled for you. That's on purpose, removing v1 is one of the very last step. The two can coexist and can be tested on prod, as far as you don't deploy v2 frontend.

I did not attempt to rework how events and status updates are handled. I did that twice in the past and it was never merged, I guess because it's complicated. I identified 3 points to hook v2 into. This is one of those.